src/EnergyPlus/UnitarySystem.cc

```diff
@@ -11669,6 +11846,8 @@ namespace UnitarySystems {
         } break;
         case DataHVACGlobals::Coil_CoolingWaterToAirHPSimple: {
             // WSHP coils do not have a PLF curve and do not adjust fan performance
+            bool errFlag = false;
+            this->heatPumpRunFrac(PartLoadRatio, errFlag, this->m_WSHPRuntimeFrac);
```

Comment:
@lgu1234 FYI. This method will no longer be used after addressing #9966. I know this solves the issue you were facing originally, but I'm more flagging this because it will cause merge conflicts for whichever branch gets merged last.

Reply:
@nealkruis Thanks for your information. I am OK to use either fix. If you fix your issues, it is better to test my new input file to ensure no overheat temperature at the outlet node. Here is the input file in this branch:
DOAToWaterToAirHPInlet_MultispeedFan.idf.